### PR TITLE
tweak sample pipeline template and add to oc cluster up

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
this cannot merge until after https://github.com/openshift/jenkins/pull/116 merges.

**Comment by bparees** [comment]
@smarterclayton @jwforres ptal, adds the sample pipeline template to oc cluster up, and some other cleanup.  also switches to use the official jenkins centos image instead of the dev tag, so we need to wait until my other PR merges before merging this.

File: examples/jenkins/pipeline/samplepipeline.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "ruby-helloworld-sample",
+    "name": "sample-pipeline",
```
Comment (smarterclayton) [inline examples/jenkins/pipeline/samplepipeline.json]: jenkins-pipeline-example

File: examples/jenkins/pipeline/samplepipeline.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
     "creationTimestamp": null,
     "annotations": {
-      "description": "This example shows how to create a simple ruby application in openshift origin v3",
+      "description": "This example uses a pipeline to build and deploy a simple ruby application in openshift origin v3",
```
Comment (smarterclayton) [inline examples/jenkins/pipeline/samplepipeline.json]: "This example showcases the new Jenkins Pipeline integration into OpenShift to perform continuous integration and deployment right from the platform.  Within the sample application is a Jenkinsfile - a definition of a multi-stage CI process that leverages the underlying OpenShift platform for dynamic and scalable builds.  OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle.  For more information, see <LINK>"

File: examples/jenkins/pipeline/samplepipeline.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
       "iconClass": "icon-ruby",
```
Comment (smarterclayton) [inline examples/jenkins/pipeline/samplepipeline.json]: Use the jenkins icon.

File: examples/jenkins/pipeline/samplepipeline.json (hunk 2 above)
Comment (smarterclayton) [inline examples/jenkins/pipeline/samplepipeline.json]: @jwforres help wordsmith this.

Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: the template isn't in the sample app, it's actually coded into the template.

"This example showcases the new Jenkins Pipeline integration in OpenShift to perform continuous integration and deployment right from the platform. Within the sample is a Jenkinsfile - a definition of a multi-stage CI process that leverages the underlying OpenShift platform for dynamic and scalable builds. OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle."

What did you have in mind for the "for more information" reference?  official docs?  more description of this specific sample (the readme)?

File: examples/jenkins/pipeline/samplepipeline.json (hunk 1 above)
Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: k

File: examples/jenkins/pipeline/samplepipeline.json (hunk 3 above)
Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: reluctant k

File: examples/jenkins/pipeline/samplepipeline.json (hunk 2 above)
Comment (jwforres) [inline examples/jenkins/pipeline/samplepipeline.json]: Just some tweaks

"This example showcases the new Jenkins Pipeline integration in OpenShift, which performs continuous integration and deployment right on the platform. The template contains a Jenkinsfile - a definition of a multi-stage CI process - that leverages the underlying OpenShift platform for dynamic and scalable builds. OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle in a single view."

**Comment by bparees** [comment]
@smarterclayton @jwforres updates pushed

**Comment by bparees** [comment]
this PR can now merge once it gets a signoff

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
(failures were known flakes that have been fixed)

**Pull request merged**